### PR TITLE
Add Travis configuration.

### PR DESCRIPTION
**Pull request by Coneko**
Adds a Travis configuration so the pod is tested.

### CODE REVIEW

**Comment by Coneko** [comment]
Trying to add coverage.

File: test/test-fs-event.c [hunk 1]
```diff
@@ -263,7 +263,7 @@ TEST_IMPL(fs_event_watch_dir) {
   ASSERT(r == 0);
   r = uv_timer_init(loop, &timer);
   ASSERT(r == 0);
-  r = uv_timer_start(&timer, fs_event_create_files, 100, 0);
+  r = uv_timer_start(&timer, fs_event_create_files, 250, 0);
```
Comment (Coneko) [inline test/test-fs-event.c:266]: Had to increase the timeout because Travis is slow.

File: test/test-fs-event.c [hunk 2]
```diff
@@ -362,7 +362,7 @@ TEST_IMPL(fs_event_watch_file_current_dir) {
   r = uv_timer_init(loop, &timer);
   ASSERT(r == 0);
 
-  r = uv_timer_start(&timer, timer_cb_touch, 1, 0);
+  r = uv_timer_start(&timer, timer_cb_touch, 2, 0);
```
Comment (Coneko) [inline test/test-fs-event.c:365]: Had to increase the timeout because Travis is slow.

**Comment by Coneko** [comment]
Example test runs at https://travis-ci.org/Coneko/libuv/branches
Example coverage https://coveralls.io/jobs/5520966

**Comment by ide** [comment]
Thanks for the PR. Do you think you could submit most of these changes to the upstream repo at https://github.com/libuv/libuv? They may be interested in improving the robustness of their test runner setup. I want to keep this repo mostly specific to the CocoaPods release (the podspec) and would take the .travis.yml file if they don't, but want to avoid maintaining a fork.

**Comment by Coneko** [comment]
Makes sense, thanks for the quick reply.

I've sent out libuv/libuv#321.

**Pull request closed without merging**